dbinspector/src/main/kotlin/com/infinum/dbinspector/DbInspector.kt

```diff
@@ -8,22 +8,22 @@ import timber.log.Timber.DebugTree
 
 public object DbInspector {
 
-    private var loggerPlanted = false
-
-    @JvmStatic
-    public fun show() {
-
-        if (BuildConfig.DEBUG && !loggerPlanted) {
-            loggerPlanted = true
+    init {
+        if (BuildConfig.DEBUG) {
             Timber.plant(DebugTree())
         }
+    }
```

Comment:
@jakoss Basically, I changed only this code to avoid a Boolean flag. Can you please test this and confirm its OK?

Reply:
No idea why i didn't come up with that :)